Program.cs

```diff
@@ -1,7 +1,32 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿using System.Runtime.CompilerServices;
+
+internal class Program {
+   private static void Main (string[] args) {
+      int a = 1, z = 100, guess;
+      Console.Write ($"Guess random number from {a} to {z} and press enter key.");
+      Console.ReadKey ();
+      int attempt = 1;
+      while (true) {
+         guess = (a + z) / 2;
+         char key;
+         Console.Write ($"\nIs {guess} your answer?(Y/N): ");
+         key = Console.ReadKey ().KeyChar;
+         if (key == 'Y') {
+            Console.Write ($"\n{guess} is your guessed random number and I found it in {attempt} attempt.");

```

Comment:
Can you use [Ternary Operator](https://www.w3schools.com/cs/cs_conditions_shorthand.php) to print either "attempt" or "attempts" to make it grammatically correct?